databases/week1/queries.sql

```diff
@@ -0,0 +1,10 @@
+SELECT COUNT(*) FROM task;
```

Comment:
Nice! Also, you could assign a neater name to the COUNT() column by using the following pattern after the COUNT() expression:

`AS "Sample name here"`